tutorials/gui_for_python_applications/README.md

```diff
@@ -0,0 +1,187 @@
+# Adding a GUI to Holoscan Python Applications
```

Comment:
Please add a `metadata.json` file to describe the tutorial (see [template](https://github.com/nvidia-holoscan/holohub/tree/main/tutorials/template))